### PR TITLE
Print error on outdated DB migration version

### PR DESCRIPTION
**Pull request by natiiix**
# Description

If the DB version is not the latest, the aggregator service will not start.

## Type of change

- New feature (non-breaking change which adds functionality)

## Testing steps

`make before_commit`

### CODE REVIEW

File: aggregator.go [hunk 1]
```diff
@@ -105,6 +105,30 @@ func closeStorage(storage *storage.DBStorage) {
 	}
 }
 
+func prepareDBMigrations(dbStorage *storage.DBStorage) int {
+	// This is only used by some unit tests.
+	if autoMigrate {
```
Comment (tisnik) [inline aggregator.go:110]: do we need to use global var there?

Comment (natiiix) [reply aggregator.go:110]: @tisnik The only alternative I could think of was adding a field into the config file, but that felt like even more effort.

Comment (tisnik) [reply aggregator.go:110]: I meant that `autoMigrate` might be parameter passed into the function.

Comment (tisnik) [reply aggregator.go:110]: I don't want to stop your work and this one is very minor thing

**Pull request merged**